### ISSUE TITLE
Revision token, key rotation

### ISSUE THREAD

**Issue by mikehelmick** [google/exposure-notifications-server#791]
### TL;DR

Revision token encryption keys are managed at the DB level.
It would be good to periodically rotate these keys and then age out old keys (thus rendering useless old revision tokens).

Currently thinking to rotate keys ever 8 days and keep the most recent 2 active.

The necessary plumbing for key rotation is already built in
https://github.com/google/exposure-notifications-server/blob/main/internal/revision/database/revision.go

### Design

**Proposal**

Build this into an existing periodic job, i.e. the cleanup job, or create a new period job specifically for key rotation.
In either case, the job will need encryptor/decryptor permissions to wrap the newly generated key for storage.

The existing locking mechanism should be used so that in a race condition, two jobs to not simultaneously rotate the key. Doing so wouldn't be the worst thing in the world, but you would have more keys active than necessary.

Keys that have been active for more than 15 days should be destroyed. This will make it so the sever can no longer decrypt these older revision keys.

**Alternatives considered**

Considered not rotating keys, but certainly not a best practice.

Can't use auto KMS rotation (or could on top of this) because this is done w/ envelope encryption.

/kind enhancement
/kind good-first-issue
